assets/js/modules/adsense/util/status.js

```diff
@@ -134,8 +134,9 @@ export const determineSiteStatus = ( {
 		return undefined;
 	}
 
+	const lowerSiteURL = siteURL.toLowerCase();
 	const hasSiteURL = urlChannels.some( ( urlChannel ) => {
-		return 0 <= siteURL.indexOf( urlChannel.urlPattern );
+		return 0 <= lowerSiteURL.indexOf( urlChannel.urlPattern );
```

Comment:
Let's use lower cased `urlPattern` as well:

```suggestion
		return 0 <= lowerSiteURL.indexOf( urlChannel.urlPattern.toLowerCase() );
```